base/challenge-skeleton/README.md

```diff
@@ -0,0 +1,40 @@
+# Quickstart guide to writing a challenge:
+
+* Put your challenge binary under challenge/image/ and update challenge/image/Dockerfile.
+* By default /chroot/home/user/chal will be run. The command line can be changed in challenge/runtime-files/nsjail.cfg.
+* Try the challenge locally with `make test-docker`.
+* If you have a cluster ready, try it remotely with `make start && make port-forward`.
+
+## healthcheck
+
+* The healthcheck needs to run a webserver on port 8080 that responds to /healthz requests.
```

Comment:
Will the skeleton provide an example web server?

Maybe we can make the port something else, like 911 or something to make it memorable as the healthcheck (I forgot 8080 was taken, and had issues with the Apache example)